cmd/agent/gui/systray/Sources/gui.swift

```diff
@@ -225,11 +238,11 @@ class AgentManager {
         let currentLoginStatus = getLoginStatus()
         var command: String
         if currentLoginStatus { // enabled -> disable
-            command = "delete every login item whose name is \"Datadog Agent\""
+            command = "/bin/launchctl unload -w " + userAgentPlistPath
```

Comment:
```suggestion
            // `unload` stops a service running for the current user session, the `-w` flag will disable it going forward
            command = "/bin/launchctl unload -w " + userAgentPlistPath
```

suggestion: clarifying comment on the specifics of launchctl and the -w flag